nginx/src/css/settings.css

```diff
@@ -163,7 +163,7 @@
 	justify-content: center;
 	align-items: center;
 	z-index: 1000;
-	display: none;
+	display: hidden;
```

Comment:
display `hidden` is not a valid display field

Reply:
adressed in : [6049901](https://github.com/TheSecretOrganization/ft_transcendence/pull/247/commits/6049901c8f4d3865f0c8b2a8c77bd795a1b6e960)